jujugui/static/gui/src/app/components/scale-service/_scale-service.scss

```diff
@@ -0,0 +1,51 @@
+.scale-service {
+
+    input[type=text] {
+        margin: 20px;
+        height: 28px;
+    }
+
+    &__units {
+        position: relative;
+
+        input[type=text] {
+            background-color: transparent;
```

Comment:
The background of the element is white I need it to be transparent so that the 'units' that is placed behind it is visible. This is required because 'placeholder' places it on the left.